forum/models/contents.py

```diff
@@ -17,6 +17,16 @@ class BaseContents(MongoBaseModel):
     content_type: str = ""
     COLLECTION_NAME: str = "contents"
 
+    def get(self, _id: str) -> Optional[dict[str, Any]]:
```

Comment:
we don't need to implement this, it's already implemented in the base class.

Reply:
yes, removed.